src/ert/gui/ertnotifier.py

```diff
@@ -1,28 +1,30 @@
-from typing import Optional
+from typing import List, Optional
 
 from qtpy.QtCore import QObject, Signal, Slot
 
-from ert.storage import Ensemble, Storage
+from ert.storage import LocalEnsemble, LocalExperiment, LocalStorage
 
 
 class ErtNotifier(QObject):
     ertChanged = Signal()
     storage_changed = Signal(object, name="storageChanged")
+    current_experiment_changed = Signal(object, name="currentExperimentChanged")
     current_ensemble_changed = Signal(object, name="currentEnsembleChanged")
 
     def __init__(self, config_file: str):
         QObject.__init__(self)
         self._config_file = config_file
-        self._storage: Optional[Storage] = None
-        self._current_ensemble: Optional[Ensemble] = None
+        self._storage: Optional[LocalStorage] = None
+        self._current_experiment: Optional[LocalExperiment] = None
```

Comment:
For this and `current_ensemble`, store a direct reference to the object instead of an ID (+ some `@property` fns that fetch from self._storage based on this), would be a bit more decoupled than storing a direct reference